apps/a11y-tests/scripts/generate-sarif-reports.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs');
+const path = require('path');
+const mkdirp = require('mkdirp');
+const { getSarifReport } = require('../lib/getSarifReport');
+
+const components = [
+  {
+    name: 'Button',
+    url: '#/examples/button'
+  },
+  {
+    name: 'Textfield',
+    url: '#/examples/textfield'
+  }
+];
+
+components.forEach(component => {
```

Comment:
Would recommend encapsulating this inside an exported function.
```js
function generateSarifReports() {
  return Promise.all(components.map(component => {
    return getSarifReport(component.url).then(/*the other stuff*/);
  });
} 

module.exports = getSarifReport;
```

Reply:
This script is intended to be directly executed in Node (`node ./scripts/generate-sarif-reports.js`). The inner function `getSarifReport` is indeed exported from `src/getSarifReport.ts`.

Reply:
Got it, hadn't noticed that this one was directly executed